Server 3 volume reseize troubleshooting (#6004)
circleci/circleci-docs: jekyll/_cci2/server-3-operator-extending-internal-volumes.adoc

As we can see, the size of our directories has been increased.

> When running through these steps, if you find that the new pods **do** show the resized volumes as expected, it is still worth running the `kubectl describe` commands shown below to check. In some instances the resize will fail, but the only way to know is by viewing an event in the output from `kubectl describe`.

Author: ```suggestion
When running through these steps, if you find, as expected, that the new pods **do** show the resized volumes, it is still worth running the `kubectl describe` commands shown below to check. In some instances the resize will fail, but the only way to know is by viewing an event in the output from `kubectl describe`.
```

For postgres
[source,bash]
----
kubectl describe pvc data-postgresql-0
----

For mongodb
[source,bash]
----
kubectl describe pvc datadir-mongodb-0
----

success looks like:
```
Events:
Type    Reason                      Age   From     Message

>    ------                      ----  ----     -------

Author: ```suggestion
```

> Normal  FileSystemResizeSuccessful  19m   kubelet  MountVolume.NodeExpandVolume succeeded for volume "pvc-b3382dd7-3ecc-45b0-aeff-45edc31f48aa"

Author: @ryanwohara this formatting looks totally weird but I copied it from the jira ticket. Any ideas how I can clean it up, or is this correct?

```

Failure might look like:
```
Warning  VolumeResizeFailed  58m   volume_expand  error expanding volume "circleci-server/datadir-mongodb-0" of plugin "kubernetes.io/aws-ebs": AWS modifyVolume failed for vol-08d0861715c313887 with VolumeModificationRateExceeded: You've reached the maximum modification rate per volume limit. Wait at least 6 hours between modifications per EBS volume.
status code: 400, request id: 3bd43d1e-0420-4807-9c33-df26a4ca3f23
Normal   FileSystemResizeSuccessful  55m (x2 over 81m)  kubelet        MountVolume.NodeExpandVolume succeeded for volume "pvc-29456ce2-c7ff-492b-add4-fcf11872589f"
```

== Troubleshooting

> If you find that after following these steps, the disk size allocated to your data directories have not increased, then you may need to restart your database pods. This, however, will cause some downtime of 1-5mins as the databases restart. You can use the commands below to restart your databases.

Author: ```suggestion
If you find that after following these steps, the disk size allocated to your data directories have not increased, then you may need to restart your database pods. This will cause downtime of 1-5 mins as the databases restart. You can use the commands below to restart your databases.
```
